ngui/ui/src/utils/layouts.ts

```diff
@@ -31,6 +31,7 @@ export const MPT_ALERTS_WARNING_4 = `#F1B178`;
 
 export const MPT_ALERTS_SUCCESS_2 = `#80E1AE`;
 export const MPT_ALERTS_SUCCESS_4 = `#005838`;
+export const MPT_TERTIARY_PRIMARY = `#ebedff`;
```

Comment:
I know it does not matter but maybe for consistency sake lets keep color hash uppercased?

Reply:
done